prow/config.yaml

```diff
@@ -42,6 +42,42 @@ presubmits:
     context: pull-charts-e2e
     rerun_command: "/test pull-charts-e2e"
     trigger: "(?m)^/test( all| pull-charts-e2e),?(\\s+|$)"
+  kubernetes/community:
+  - name: pull-community-verify
+    agent: kubernetes
+    context: pull-community-verify
+    branches:
+    - master
+    rerun_command: "/test pull-community-verify"
+    trigger: "(?m)^/test( all| pull-community-verify),?(\\s+|$)"
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:0.4
+        args:
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--clean"
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/service-account/service-account.json
+        volumeMounts:
+        - name: service
+          mountPath: /etc/service-account
+          readOnly: true
+        - name: cache-ssd
+          mountPath: /root/.cache
+        ports:
+        - containerPort: 9999
+          hostPort: 9999
```

Comment:
@krzyzacy what did we decide about the port / cache for non-bazel jobs?

Reply:
And if we want the cache on this job then probably also `args: - "--git-cache=/root/.cache/git"`

Reply:
> what did we decide about the port / cache for non-bazel jobs

@spxtr @rmmh shall we just make cache can be shared with multiple jobs? Or that's already the case?